deployment/deployment.go

```diff
@@ -33,17 +33,18 @@ type Node struct {
 	TxPowerDBm       float64
 	FreqGHz          vlib.VectorF
 	Mode             TxRxMode `json:"TxRxMode"`
-	alias            int
+	Alias            int
 	Active           bool
 	RxNoiseFigureDbm float64 // NoiseFigure of the
 }
 
-func (n Node) Alias() int {
-	return n.alias
-}
+// func (n Node) Alias() int {
+// 	return n.Alias
+// }
 
-func (n *Node) SetAlias(a int) {
-	n.alias = a
+func (n Node) SetAlias(a int) {
+	n.Alias = a
+	// fmt.Println("updated alias, vid:", n.alias, a)
 }
```

Comment:
why expose the Alias variable ? when there is function to access it

Reply:
Changed it back to local. Wasn't working initially that's why turned it to global.